src/bridging/providers/MockBridgeProvider.ts

```diff
@@ -0,0 +1,179 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import { Signer } from 'ethers'
+import {
+  BridgeDeposit,
+  BridgeHook,
+  BridgeProvider,
+  BridgeProviderInfo,
+  BridgeQuoteResult,
+  BridgeStatus,
+  BridgeStatusResult,
+  GetBuyTokensParams,
+  QuoteBridgeRequest,
+} from '../types'
+import { ChainInfo, EthereumLogo, mainnet, sepolia } from '../../common'
+import { EvmCall } from '../../common/transaction'
+import { TokenInfo } from '../../common/tokens'
+
+const BRIDGING_ID = '123456789asdfg'
+const MOCK_TX: EvmCall = {
+  to: '0x0000000000000000000000000000000000000001',
+  data: '0x0',
+  value: '0x0',
+  isDelegateCall: true,
+}
+
+//See https://github.com/wevm/viem/blob/main/src/chains/definitions/optimism.ts
+const optimism: ChainInfo = {
+  id: 10,
+  name: 'optimism',
+  label: 'Optimism',
+  logo: { light: EthereumLogo, dark: EthereumLogo },
+  nativeCurrency: {
+    chainId: 10,
+    address: '0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2',
+    decimals: 18,
+    name: 'Wrapped Ether',
+    symbol: 'WETH',
```

Comment:
Nitpick: I know it's mocked, but even, the native currency is ETH rather than WETH

```suggestion
    address: '0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee,
    decimals: 18,
    name: 'Ether',
    symbol: 'ETH',
```